### PR TITLE
added EB ambiguity

### PR DESCRIPTION
**Pull request by pseyfert**
feel free to edit, eg. for linking styles

### CODE REVIEW

File: glossary/E.md [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+Abbreviation for:
+
+ * Editorial Board
+ * Event Builder
+
+## Editorial Board
 
 Group responsible for ensuring that all LHCb publications are consistent with the standards and style of the collaboration.
 
+## Event Builder
```
Comment (eduardo-rodrigues) [inline glossary/E.md]: Is there a link to a webpage we can add here?

Comment (pseyfert) [reply glossary/E.md]: good question … maybe section 3.5 of https://cds.cern.ch/record/1701361 ?

Comment (eduardo-rodrigues) [reply glossary/E.md]: I see no better reference right now. Indeed the link at https://lbtwiki.cern.ch/bin/view/Online/WebHome does not give anything worth linking to. I would go with your ref. Ready to be merged, I reckon, module the addition of the link. Thanks.

**Comment by eduardo-rodrigues** [comment]
@pseyfert, shall we get to the bottom of this PR? Just a little extra do do.

**Comment by pseyfert** [comment]
just pushed with the link to the trigger and online upgrade tdr

**Pull request merged**